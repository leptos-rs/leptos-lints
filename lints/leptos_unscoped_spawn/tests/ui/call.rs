use leptos::task;
use leptos::task::spawn_local;

fn main() {
    spawn_local(async {
        // ...
    });

    task::spawn_local(async {
        // ...
    });

    leptos::task::spawn_local(async {
        // ...
    });
}
