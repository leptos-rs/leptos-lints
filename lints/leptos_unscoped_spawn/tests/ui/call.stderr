error: use of `leptos::task::spawn_local`
  --> $DIR/call.rs:2:19
   |
LL | use leptos::task::spawn_local;
   |                   ^^^^^^^^^^^
   |
   = help: prefer `leptos::task::spawn_local_scoped`. For further information visit https://github.com/leptos-rs/leptos-lints/tree/main/lints/leptos_unscoped_spawn#readme
   = note: `#[deny(leptos_unscoped_spawn)]` on by default

error: use of `leptos::task::spawn_local`
  --> $DIR/call.rs:5:5
   |
LL |     spawn_local(async {
   |     ^^^^^^^^^^^
   |
   = help: prefer `leptos::task::spawn_local_scoped`. For further information visit https://github.com/leptos-rs/leptos-lints/tree/main/lints/leptos_unscoped_spawn#readme

error: use of `leptos::task::spawn_local`
  --> $DIR/call.rs:9:11
   |
LL |     task::spawn_local(async {
   |           ^^^^^^^^^^^
   |
   = help: prefer `leptos::task::spawn_local_scoped`. For further information visit https://github.com/leptos-rs/leptos-lints/tree/main/lints/leptos_unscoped_spawn#readme

error: use of `leptos::task::spawn_local`
  --> $DIR/call.rs:13:19
   |
LL |     leptos::task::spawn_local(async {
   |                   ^^^^^^^^^^^
   |
   = help: prefer `leptos::task::spawn_local_scoped`. For further information visit https://github.com/leptos-rs/leptos-lints/tree/main/lints/leptos_unscoped_spawn#readme

error: aborting due to 4 previous errors

